packages/components/src/mobile/global-styles-context/utils.native.js

```diff
@@ -191,6 +190,22 @@ export function getBlockTypography(
 	return typographyStyles;
 }
 
+/**
+ * Return a value from a certain path of the object.
+ * Path is specified as an array of properties, like: [ 'parent', 'child' ].
+ *
+ * @param {Object} object Input object.
+ * @param {Array}  path   Path to the object property.
+ * @return {*} Value of the object property at the specified path.
+ */
+const getValueFromObjectPath = ( object, path ) => {
+	let value = object;
+	path.forEach( ( fieldName ) => {
+		value = value?.[ fieldName ];
```

Comment:
We're abstracting to a separate function because this is needed twice in this component and because it IMHO improves readability. 

Optional chaining here will help us handle nullish values both at the first level and nested inside objects or arrays, making the function essentially as save as `get()` was.